brickflow/codegen/databricks_bundle.py

```diff
@@ -698,6 +698,21 @@ def proj_to_bundle(self) -> DatabricksAssetBundles:
         jobs = {}
         pipelines = {}  # noqa
         for workflow_name, workflow in self.project.workflows.items():
+            if (
+                self.env == BrickflowDefaultEnvs.LOCAL.value
+                and BrickflowEnvVars.BRICKFLOW_DEPLOY_ONLY_WORKFLOWS.value in os.environ
```

Comment:
should we not throw an error and skip looping through if `deploy_only_workflows` is true and `env` is not local?

Reply:
good catch.. updated